docs/api-docs/useful-info/api-data-sources-availability.mdx

```diff
@@ -99,8 +107,41 @@ Below are a list of known issuses with our data sources:
 - No container weight
 - Only Dry container types are mapped to our system
 
+## Rail Coverage
```

Comment:
I think we should omit this until we're out of beta - this is already changing and may add confusion

Reply:
Do we know which parts will change and which will stay the same?  I'd imagine it would be really useful for customers to be able to distinguish between "don't have yet" vs "rail carrier doesn't provide"

Reply:
TODO: Remove

Reply:
Removed ✅